frontend/src/components/constants/index.tsx

```diff
@@ -1,5 +1,14 @@
 export const CURRENCY_DOLLAR = "$";
-
 export const SHIPPING_PATH = "/shipping";
 export const SIGNIN_PATH = "/user/signin";
+export const SIGNUP_PATH = "/user/signup"
 export const HOME_PATH = "/"
+export const SIGNUP_TEXT = "Signup!"
+export const SIGNIN_TEXT = "Signin!"
+export const SIGNOUT_TEXT = "Sign out!"
+export const SHOPPING_CART_PATH = "/cart"
+export const UPDATE_USER_PATH = "/user/update"
+export const ORDERS_HISTORY_PATH = "/ordershistory"
+export const PROFILE_TEXT = "Profile"
+export const MY_ORDERS_TEXT = "My Orders"
+export const CART_TEXT = "Cart"
```

Comment:
Nice!!
let's order them by topic (text, path, currency ...) to make it pretty ✨